var/spack/repos/builtin/packages/meme/package.py

```diff
@@ -35,9 +35,13 @@ class Meme(AutotoolsPackage):
 
     version('4.11.4', '371f513f82fa0888205748e333003897')
 
+    variant('mpi', default=True, description='Enable MPI support')
+
     depends_on('zlib', type=('link'))
     depends_on('libxml2', type=('link'))
     depends_on('libxslt', type=('link'))
     depends_on('libgcrypt', type=('link'))
     depends_on('perl', type=('build', 'run'))
     depends_on('python@2.7:', type=('build', 'run'))
+    # openmpi support
+    depends_on('openmpi@2:', type=('link'))
```

Comment:
It seems with MPI variant set as default MPI will always be built even when +serial -mpi ~mpi are specified.

For that reason I think the latest commit would be appropriate since serial is a valid option for users with workloads that demand fat nodes.